### DISCUSSION TITLE
Topology Views

### DISCUSSION THREAD

**Discussion by FGronostaj** [netbox-community/netbox#18069] [Help Wanted!]
Hi, what do I need to do because in the topology map it doesn't show me any device
![u8s2lZWVKT](https://github.com/user-attachments/assets/fbc916df-f86b-4b92-aaf9-7a0a56ff5a15)

**Comment by betadrome** [comment]
> Hi, what do I need to do because in the topology map it doesn't show me any device ![u8s2lZWVKT](https://private-user-images.githubusercontent.com/37020631/388905354-fbc916df-f86b-4b92-aaf9-7a0a56ff5a15.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIyODYyMzYsIm5iZiI6MTczMjI4NTkzNiwicGF0aCI6Ii8zNzAyMDYzMS8zODg5MDUzNTQtZmJjOTE2ZGYtZjg2Yi00YjkyLWFhZjktN2EwYTU2ZmY1YTE1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTIyVDE0MzIxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMxYzY4ZjJmMzkwMmM5YmNjMzlkOGE1MDAwMjRlNDRjNGM5NmYzNmEwNzkzYzM2OTBkOTNlODAwYzllZGIyOGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Fl2mDrlCJkirhHMqY7kAtStwUSbPzFhmY6V5n37FW0w)

Try to use the checkboxes "show cables" and "show logical connections"
